app/helpers/katello/katello_url_helper.rb

```diff
@@ -18,6 +18,7 @@ def file_prefix?(url)
     private
 
     def valid_for_prefixes(url, prefixes)
+      return false unless URI.parse(url).host.present?
       return false unless (scheme = URI.parse(url).scheme).present?
```

Comment:
I like separate conditions here due to the returns and not a general if
block.
On Dec 2, 2015 3:16 PM, "Chris Peters" <notifications@github.com> wrote:

> In app/helpers/katello/katello_url_helper.rb
> <https://github.com/Katello/katello/pull/5640#discussion_r46469197>:
>
> > @@ -18,6 +18,7 @@ def file_prefix?(url)
> >      private
> >
> >      def valid_for_prefixes(url, prefixes)
> > +      return false unless URI.parse(url).host.present?
> >        return false unless (scheme = URI.parse(url).scheme).present?
>
> @johnpmitsch <https://github.com/johnpmitsch>
>
> I'm not sure I agree with moving these to the same line. They are checking
> different parts of the URI. Also there is a nested assignment. This gets
> messy and hard to parse when reading back over the code, imo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5640/files#r46469197>.
>